millepede.spec

```diff
@@ -1,4 +1,9 @@
+<<<<<<< HEAD
```

Comment:
@skeshri you need to solve the merge conflict and test it yourself before creating a new PR. (also, the difference between the two version is 2 numbers of the version, you can easily modify them without merging or cherry-picking...)